docs/meta/ticket_process.md

```diff
@@ -0,0 +1,84 @@
+This doc is here to explain the lifecycle of a Beaker ticket.  If you have any
+questions about the workflow of a Beaker ticket, or the stage that it's in, you
+should be able to answer them here.  If not, then please let QE know, and we'll
+work to answer your question and update this doc, so that we can do this better
+going forward.
+
+Note that a typical Beaker ticket goes through these states. They can sometimes
+go in order, but on average, they loop around through the various stages,
+can cycle a number of times before becoming resolved, and will sometimes skip
+stages as well.
+
+## Administrivia
+
+- Beaker tickets live in the 
+[BKR project](https://tickets.puppetlabs.com/issues/?jql=project%20%3D%20BKR)
+- Create a [Jira account](http://tickets.puppetlabs.com) to interact with them 
+
+## Pre-Development States
+
+These are the states that a Beaker ticket goes through before it is picked up for
+development. Note that this is only for work done by the Beaker team. If there
+is something you'd like to contribute to, we would appreciate that you go
+through the development states for tracking purposes, but you can skip directly
+to the "In Progress" state if you're going to work on a ticket.
+
+- Open
```

Comment:
The way this formats looks kind of weird; the bullet becomes somewhat indented, but the text is still left-aligned; maybe use the `###` instead?